go/enclave/components/batch_executor.go

```diff
@@ -731,26 +731,12 @@ func (executor *batchExecutor) verifySyntheticTransactionsSuccess(transactions c
 }
 
 func (executor *batchExecutor) executeTx(ec *BatchExecutionContext, tx *common.L2PricedTransaction, offset int, noBaseFee bool) (*core.TxExecResult, error) {
-	vmCfg := vm.Config{
-		NoBaseFee: noBaseFee,
-	}
 	ethHeader := *ec.EthHeader
 	before := ethHeader.MixDigest
 	ethHeader.MixDigest = executor.entropyService.TxEntropy(before.Bytes(), offset)
 
 	// if the tx fails, it handles the revert
-	txResult := evm.ExecuteTransaction(
-		tx,
-		ec.stateDB,
-		&ethHeader,
-		ec.Chain,
-		ec.ChainConfig,
-		ec.GasPool,
-		ec.usedGas,
-		vmCfg,
-		offset,
-		executor.logger,
-	)
+	txResult := executor.evmFacade.ExecuteTx(tx, ec.stateDB, &ethHeader, ec.GasPool, ec.usedGas, offset, noBaseFee)
```

Comment:
No need to change but the one liners are hard to view on one monitor when splitting tabs and AI can change something subtly